src/routes/project/TaskRow.svelte

```diff
@@ -144,4 +175,19 @@
 		border-radius: 50%;
 		overflow: hidden;
 	}
+	.sticky-name {
+		position: absolute;
+		top: auto;
+		left: 0;
+		padding: var(--padding-bars) 2rem 0;
+		height: auto;
+		max-width: 100%;
+		display: inline-block;
+		overflow: hidden;
+		text-overflow: ellipsis;
+	}
+	.sticky-name_right {
```

Comment:
Again, this should be a modifier, if we stick to the BEM convention, so, `class="task__detail-name task__detail-name--sticky task__detail-name--sticky-right"`. Yes, BEM can be intimidating. 😄